docker/Dockerfile

```diff
@@ -25,27 +25,27 @@ RUN pip uninstall -y typing
 ENV DJANGO_SETTINGS_MODULE="codecov.settings_enterprise"
 # Remove the settings dev and enterprise files.
 # These should *never* make it to enterprise.
-RUN rm /app/codecov/settings_dev.py && \
-    rm /app/codecov/settings_prod.py && \
-    rm /app/codecov/settings_test.py && \
-    rm /app/codecov/settings_staging.py && \
-    rm /app/dev.sh && \
-    rm /app/migrate.sh && \
-    rm /app/prod.sh && \
-    rm /app/staging.sh && \
-    rm /app/production.yml && \
-    rm /app/development.yml
+RUN rm /app/apps/codecov-api/codecov/settings_dev.py && \
```

Comment:
Optional: might be worth adding a const or something for this value to explain what it is